cibuildwheel.toml

```diff
@@ -0,0 +1,10 @@
+[tool.cibuildwheel]
+skip = ["pp*", "*-musllinux*"]
+test-extras = "test"
+test-command = [
+    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/runner.py",
+    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/test_memleaks.py"
+]
+
+[tool.cibuildwheel.macos]
+archs = ["x86_64", "arm64"]
```

Comment:
Can we do this without using an additional file? I like having the test runner defined in the main file.
If we can't avoid this, at the very least can we put this file under `.github/workflows/` directory instead of the root directory?

Reply:
> Can we do this without using an additional file?

It could be done but I'd rather not.
cibuildwheel is more than just a CI tool & as such, I see its config file living at project root just like the ones from flake8/isort/coverage.
It's more & more common for cibuildwheel users to put there config in `pyproject.toml` and be able to just run `cibuildwheel`, whatever the CI provider, or run locally (which I use extensively) to reproduce the steps that would happen on CI.
Running locally, I can just do `cibuildwheel --platform macos --config-file cibuildwheel.toml` rather than remembering/typing all of the CI config from the GitHub workflow in my console.
This file is CI provider independent so if it were to be kept & moved, would you be okay to move it to `.ci/` ?
Another option if you're open to it would be to add a `pyproject.toml` file with cibuildwheel config inside. It would simplify the workflow even more (& running locally) by removing the need to specify a config file:
```toml
[build-system]
requires = ["setuptools>=43"]
build-backend = "setuptools.build_meta"

[tool.cibuildwheel]
skip = ["pp*", "*-musllinux*"]
test-extras = "test"
test-command = [
    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/runner.py",
    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/test_memleaks.py"
]

[tool.cibuildwheel.macos]
archs = ["x86_64", "arm64"]
```

Reply:
> Another option if you're open to it would be to add a pyproject.toml file with cibuildwheel config inside.

Interesting. I had no idea. I think this is the best course, also because it seems natural to move flake8 / autopep8, isort and coveragerc configs in there as well as a next step.